bin/_blackbox_common.sh

```diff
@@ -277,6 +277,9 @@ function shred_file() {
     #NOTE: srm by default uses 35-pass Gutmann algorithm
     CMD=srm
     OPT=-f
+  elif _F=$(mktemp); rm -P "${_F}" >/dev/null 2>/dev/null ; then
```

Comment:
And of course it needs to go after the flags, which it turns out it already does on [line 289](https://github.com/grahamlyons/blackbox/blob/19ec871f0cfe9b99e00d15e6e0679beab73935b6/bin/_blackbox_common.sh#L289). It took me a few minutes to clock and I've had coffee...